### DISCUSSION TITLE
Focus Property-Name field after adding a "Applied property" on API-Keys

### DISCUSSION THREAD

**Discussion by Sebbl22** [datalust/seq-tickets#1198] [Ideas]
Hello,
it would be nice, to focus the property name field after adding a property to an API-Key instead of the property value field, so a large number of "Applied properties" can be added easily. 
Here is a GIF for better visualization:
![Animation](https://user-images.githubusercontent.com/10319545/115883700-5e370300-a44e-11eb-94b7-df75ed089304.gif)

**Comment by nblumhardt** [comment]
Thanks! That make sense, we'll give this a shot 👍
